src/test/scala/com/target/data_validator/validator/RangeCheckSpec.scala

```diff
@@ -266,13 +282,20 @@ class RangeCheckSpec extends FunSpec with Matchers with TestingSparkSession {
         ))
       }
 
+      it("threshold works as expected") {
+        val maxJson = Json.fromDouble(3.0)
+        val sut = RangeCheck("max",
+          None, maxJson, None, Some("30%")) // scalastyle:ignore
+        assert(true) // FIXME!!!!!
```

Comment:
do we need this?

Reply:
nope, We have other tests that check the calculation of threshold count.